web/src/pm_list.js

```diff
@@ -218,4 +253,8 @@ export function initialize() {
 
         zoom_out();
     });
+
+    $("body").on("input", "#filter-pm-input", () => {
+        update_private_messages();
```

Comment:
Let's add `e.stopPropagation();` and `e.preventDefault();` here.

Reply:
Updated!